tools/release/common.sh

```diff
@@ -44,9 +44,9 @@ find_prev_release() {
   find_last_release
```

Comment:
find_last_release also has `--sort=taggerdate` , which seems to be also wrong when we're doing parallel releases due to patching :(

Reply:
I will resolve this because this conversation is not relevant anymore.